### DISCUSSION TITLE
MegalLinter 8.2.0 -- little confusion about PHPCS

### DISCUSSION THREAD

**Discussion by llaville** [oxsecurity/megalinter#4265] [General]
The recent release https://github.com/oxsecurity/megalinter/releases/tag/v8.2.0 tells us that `APPLY_FIXES` was activated for PHPCS

It's for PHP-CS-Fixer and not PHP_CodeSniffer !

**Comment by nvuillam** [comment]
Fixed :)
Next time you'll think to input the CHANGELOG 😇
